Issue 5 - December 2019 Content Update

feaselkl opened the pull request:
Resolve problems brought up in Issue 4, Issue 5, and Issue 7 around the hands-on lab.

Rewrote the Before the Lab guide to use the Azure Data Studio deployment wizard.  This resolves Issue 4 along the way.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/MCW-Modernizing-Data-Analytics-with-SQL-Server-2019?pullRequest=8) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/MCW-Modernizing-Data-Analytics-with-SQL-Server-2019?pullRequest=8) before we can accept your contribution.<br/><br/>:x: feaselkl [sign now](https://cla.opensource.microsoft.com/microsoft/MCW-Modernizing-Data-Analytics-with-SQL-Server-2019?pullRequest=8)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/MCW-Modernizing-Data-Analytics-with-SQL-Server-2019?pullRequest=8) it.</sub>

feaselkl commented:
@IkeEllis here are my changes.  Please tag anyone else you'd like to review them and let me know what you would like for me to change.

feaselkl commented:
I have also updated the batch files which are used in Task 4 of the Before the HOL document.  I added the file bootstrap-sample-db.cmd to the Resources set and referenced it; that way I could update the script.  I also updated upload-sample-files.cmd to match.  The major change there was to change sa to admin for the default username.

timahenning commented:
@feaselkl Hey Kevin!  Thank you for submitting your PR.  Ike confirmed this morning he reviewed your PR and merged your changes into his PR.

Tim
